projects/dsp-ui/src/lib/viewer/values/color-value/color-picker/color-picker.component.scss

```diff
@@ -1 +1,6 @@
 @import "../../../../../assets/style/viewer";
+
+::ng-deep .mat-form-field-flex,
```

Comment:
instead of using `::ng-deep`, would it be possible to do it in one of the main stylesheets (src/assets/style/*)?

Reply:
This is what I always suggested. I can move it to ds-ui.scss...

Reply:
If you move this to dsp-ui.scss, then this will be applied to every .mat-form-field-flex in the DOM. I think I originally tried to do this in dsp-ui.scss and select it via a class name but it wasn't working.

Reply:
@mdelez Yes I know. But I will use the component tag name as first selector then it should work. I'm not sure...I will try

Reply:
I'm actually dealing with this same CSS scenario in this [PR](https://github.com/dasch-swiss/dsp-ui-lib/pull/198). I've found that within dsp-ui.scss, I can select custom class names as expected. However the issue arrises when I try to do the same thing in _viewer.scss. Do you know why I am not able to do this in _viewer.scss?

Reply:
have you imported the _viewer file in the css file?

Reply:
Well, I feel stupid now... It works now that I've imported the _viewer file

Reply:
I use now the `:host` selector and it works the same way...

Reply:
Which version should we commit and push?

Reply:
well, shouldn't we keep it consistent with the other css files? or is it something different?

Reply:
@mdelez Do you want to push your solution? Mine is not well done, I think...

Reply:
is the decision between `:host` and `::ng-deep` in the the color value scss file? If so, I would choose `:host`.

Reply:
I pushed it, but not the thing with `:host` the host is bigger than the input and the pointer was not only inside the box.
@mike You can still override my solution

Reply:
aha, what you did looks good :)